third_party/boringssl/BUILD.gn

```diff
@@ -102,7 +102,7 @@ static_library("crypto") {
     configs -= [ "//starboard/build/config:size" ]
     configs += [ "//starboard/build/config:speed" ]
 
-    if (sb_is_modular) {
+    if (sb_is_modular && current_toolchain == cobalt_toolchain) {
```

Comment:
Can we try to remove current_toolchain == cobalt_toolchain in the //third_party BUILD.gn code?
I think they are all used when building only with the cobalt toolchain anyways.

Reply:
Done